roseus/euslisp/roseus-utils.l

```diff
@@ -55,6 +55,35 @@
     ;; FIXME:: add coordinates of camera
     (make-camera-from-ros-camera-info-aux pwidth pheight p nil)))
 
+;; Image
+(when (and
+       (assoc :create-viewer (send camera-model :methods))
+       (probe-file (format nil "~A/irteus/demo/sample-camera-model.l" *eusdir*)))
```

Comment:
@k-okada thank you for reviewing.

- how can we use `find-method` with class object to inspect instance method existence?
- Actually no condition is necessary.(we can use these functions on hydro deb env), but not tested. I check `probe-file` because test code is enabled only when `sample-camera-model.l` exists.